generators/server/templates/quarkus/src/main/java/package/web/rest/errors/EmailAlreadyUsedException.java.ejs

```diff
@@ -18,9 +18,13 @@
 -%>
 package <%=packageName%>.web.rest.errors;
 
+import java.net.URI;
+
 public class EmailAlreadyUsedException extends BadRequestAlertException {
 
+    private static final URI TYPE = URI.create(ErrorConstants.PROBLEM_BASE_URL + "/email-already-used");
```

Comment:
Can you move `TYPE` to `ErrorConstants`, please

Reply:
Sure I can, but let me first explain, why I think it's better this way: those specific exceptions are included on condition (!skipUserManagement) - if they are not included, people will see some strange **unused** constants in `ErrorConstants`. I also don't think they need to be shared, and even if - I'd prefer little duplication over coupling and making this conditional inclusion harder. 
But I may be missing some bigger picture here, so I'll happily move them back to `ErrorConstants` class :)

Reply:
🤔 I tend to try to keep consistency between JHipster Spring and Quarkus that's why I would recommend using `ErrorConstants`. But I understand your argument...
Let it as is.